### DISCUSSION TITLE
GitHub Packages

### DISCUSSION THREAD

**Discussion by mattheys** [blakeblackshear/frigate#750] [Ideas]
Hi,

Was just wondering if it was worth publishing the docker images to GitHub Packages as well as or instead of Docker Hub? Only mention it due to Docker Hub now putting a limit on the pull requests you can make (100/200) and if you have quite a few other containers then the more that can be pulled from elsewhere means that it's less likely to hit that limit. I thought that 100 would be plenty but some how managed to reach that limit myself once and had to login, not quite sure how I managed it but maybe it was watchtower in the background.

It would mean the image could be `ghcr.io/blakeblackshear/frigate:0.8.1-*` as well as `blakeblackshear/frigate:0.8.1-*`

I belive Packages is free to public repositories. I've never quite got to grips with the GitHubs Actions pipeline so not sure if it can just be setup once and left to auto build on new releases or not.

**Comment by blakeblackshear** [comment]
I will definitely do this when I build out github actions.
